magellan-library/src/main/java/com/wealthfront/magellan/lifecycle/LifecycleAwareComponent.kt

```diff
@@ -44,6 +44,14 @@ abstract class LifecycleAwareComponent : LifecycleAware, LifecycleOwner {
     return lifecycleRegistry.backPressed() || onBackPressed()
   }
 
+  fun attachToLifecycle(lifecycleAware: LifecycleAware) {
+    attachToLifecycle(lifecycleAware, LifecycleState.Destroyed)
+  }
+
+  fun removeFromLifecycle(lifecycleAware: LifecycleAware) {
+    removeFromLifecycle(lifecycleAware, LifecycleState.Destroyed)
+  }
+
```

Comment:
Why not just use default parameters for the existing attachToLifecycle methods? Wait, don't the existing attachToLifecycle methods have default parameters inherited from the interface?

If we're going to add methods like this, it should be to the `LifecycleAware` interface.

Reply:
Add a default param to LifecycleAware instead.